qlty-check/src/planner.rs

```diff
@@ -268,6 +271,17 @@ impl Planner {
             plugins: self.config.plugin.clone(),
         }));
 
+        for plugin in &self.active_plugins {
+            if let Some(tab_column_width) = plugin.plugin.tab_column_width {
```

Comment:
Rather than having this be nullable, which causes us to have to juggle Some/None, maybe we make it just always an int with a default `1` and then only add the transformer if the value is `> 1`. The reason I am suggesting this is because it seems like there is never a difference between `1` and `None`. WDYT?

Reply:
I thought using `1` might look more like a magic number, where as `None` makes it more clear that there isn't going to be a need for this directive while being more idiomatic in rust terms.

Reply:
Gonna merge this so we can release.